monitor.py

```diff
@@ -2157,6 +2165,75 @@ def suit_loadout_slots_array_to_dict(self, loadout: dict) -> dict:
 
         return slots
 
+    def _parse_navroute_file(self) -> Optional[dict[str, Any]]:
+        """Read and parse NavRoute.json."""
+        if self.currentdir is None:
+            raise ValueError('currentdir unset')
+
+        if not (pathlib.Path(self.currentdir) / 'NavRoute.json').exists():
```

Comment:
This case should be caught by the try/except in the open.  So this can be removed, right ?